database/build.gradle

```diff
@@ -7,7 +7,7 @@ var sqliteVersion = "3.45.0.0"
 dependencies {
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
     implementation "org.xerial:sqlite-jdbc:${sqliteVersion}"
-    implementation 'org.flywaydb:flyway-core:10.6.0'
+    implementation 'org.flywaydb:flyway-core:10.7.1'
```

Comment:
is this because of rebase?

Reply:
most likely, but it shouldnt show up as a new change right?

Reply:
maybe if it's based on earlier iteration of main branch, @Zabuzard prolly has some insights here.

Reply:
I just checked and the version on `develop` branch is also `10.7.1`, so this "change" is kinda invalid